apps/prairielearn/src/lib/copy-question.ts

```diff
@@ -1,104 +0,0 @@
-import * as path from 'node:path';
-
-import { type Response } from 'express';
-import fs from 'fs-extra';
-import { v4 as uuidv4 } from 'uuid';
-
-import * as error from '@prairielearn/error';
-import * as sqldb from '@prairielearn/postgres';
-import { generateSignedToken } from '@prairielearn/signed-token';
-
-import { selectCoursesWithEditAccess } from '../models/course.js';
-
-import { config } from './config.js';
-import { type Course, type Question } from './db-types.js';
-import { idsEqual } from './id.js';
-
-const sql = sqldb.loadSqlEquiv(import.meta.url);
-
-export async function setQuestionCopyTargets(res: Response) {
-  // Avoid querying for editable courses if we won't be able to copy this
-  // question anyways.
-  if (!res.locals.course.template_course && !res.locals.question.share_source_publicly) {
-    return;
-  }
```

Comment:
this auth check was removed because the checks are now done by the places that call this function.